iothub/service/src/Common/Data/AccessRights.cs

```diff
@@ -19,28 +19,28 @@ public enum AccessRights
         /// <summary>
         /// Grants read access to the identity registry.
         /// Identity registry stores information about the devices and modules permitted to connect to the IoT hub.
-        /// For more information, <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry"/>.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry"/>.
         /// </summary>
         RegistryRead = 1,
 
         /// <summary>
         /// Grants read and write access to the identity registry.
         /// Identity registry stores information about the devices and modules permitted to connect to the IoT hub.
-        /// For more information, <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry"/>.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry"/>.
         /// </summary>
         RegistryWrite = RegistryRead | 2,
 
         /// <summary>
         /// Grants access to service facing communication and monitoring endpoints.
-        /// It grants permission to send and receive device-to-cloud messages, retrieve delivery acknowledgments for sent messages
-        /// and file uploads, retrieve reported properties on device twins, update tags on device twins and desired properties,
+        /// It grants permission to receive device-to-cloud messages, send cloud-to-device messages, retrieve delivery acknowledgments for sent messages
+        /// and file uploads, retrieve desired and reported properties on device twins, update tags and desired properties on device twins,
         /// and run queries.
         /// </summary>
         ServiceConnect = 4,
 
         /// <summary>
         /// Grants access to device facing endpoints.
-        /// It grants permission to send and receive device-to-cloud messages, perform file upload from a device, receive device twin
+        /// It grants permission to send device-to-cloud messages and receive cloud-to-device, perform file upload from a device, receive device twin
```

Comment:
add  "messages" following receive cloud-to-device. But to me however the previous one seems enough as well.

Reply:
we had previously stated that this access rights grants send and receive permission for d2c devices; whereas it actually grants access rights to send d2c message and receive c2d message (notice that the source and target are different for each operation)